### PR TITLE
No config wait + IndexCheck enhancements

### PR DESCRIPTION
**Pull request by MatildaAslin**
- Sisyphus no longer waits for config file
-  Made changes to IndexCheck:
 - Modified prints
 - Fixed regex 
 - Failing indexcheck will no longer terminate sisyphus.sh. I realised that I often want to look at the quickReport even if something is found among Undetermined indices. 
- Updated test config 

**NB: For code reviewing, I suggest looking at the two first commits.**

### CODE REVIEW

File: sisyphus.pl [hunk 1]
```diff
@@ -432,7 +432,6 @@ =head1 DESCRIPTION
 check_errs \$? "bcl2fastq failed in $fastqPath/IndexCheck"
 
 checkIndices.pl -runfolder '$rfPath' -demuxSummary '$rfPath/IndexCheck/Stats' 
-check_errs \$? "IndexCheck failed"
```
Comment (b97pla) [inline sisyphus.pl]: it would be good to save the exit status and do the bailout after generating the quick report instead of just skipping the check

**Comment by b97pla** [comment]
Other than the comment and the tab-issue, I think it looks good.

**Comment by MatildaAslin** [comment]
Good point, @b97pla . I pushed a new commit now. Exit status for index check is now saved and will be checked before uploading data to Uppmax.

**Comment by b97pla** [comment]
Sorry @MatildaAslin, I lost track of this. Should it be merged?

**Comment by MatildaAslin** [comment]
Yes, it is still relevant. :)

**Pull request merged**